recipes/b2/portable/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,12 @@
+from conan import ConanFile
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch"
+    test_type = "explicit"
+
+    def build_requirements(self):
+        self.tool_requires(self.tested_reference_str)
+
+    def test(self):
+        self.run("b2 -v", run_environment=True)
```

Comment:
For the legacy generators we need to make sure the "env_info" is available and the tool is on the PATH variable (boost recipe still uses this) 1.x does not need the add reference to gain it. 2.0 generators do in oder to add the bin dirs to the build env

```suggestion
from conan import ConanFile
from conan.tools.build import can_run


class TestPackageV1Conan(ConanFile):
    settings = "os", "arch", "compiler", "build_type"

    def test(self):
        if can_run(self):
            self.run("b2 -v", run_environment=True)
```